api/webhook/taskcluster.go

```diff
@@ -122,18 +124,15 @@ func handleStatusEvent(ctx context.Context, payload []byte) (bool, error) {
 
 	// https://github.com/web-platform-tests/wpt.fyi/blob/master/api/README.md#results-creation
 	api := fmt.Sprintf("https://%s/api/results/upload", appengine.DefaultVersionHostname(ctx))
-	for browser, urls := range urlsByBrowser {
-		log.Infof(ctx, "Reports for %s: %v", browser, urls)
-		// Set timeout to 1 min (the default is 5s) to give the
-		// receiver enough time to download the reports.
-		slowCtx, cancel := context.WithTimeout(ctx, time.Minute)
-		client := urlfetch.Client(slowCtx)
-		err := createRun(client, api, username, password, urls)
-		cancel()
-		if err != nil {
-			return false, err
-		}
+
+	// Set timeout to 1 min (the default is 5s) to give the receiver enough time to download the reports.
+	slowCtx, cancel := context.WithTimeout(ctx, time.Minute)
```

Comment:
Let's abstract this `time.Minute` into a package-local variable. That way, we can write a test that verifies we get a deadline error without the test having to run for a full minute.